Assignment5/Main.cpp

```diff
@@ -0,0 +1,26 @@
+#include "Car.h"
+#include <iostream>
+
+int main() 
+{
+    Car car;
+    try 
+    {
+        car.travel();
+        std::cout << "Both people are safe." << std::endl;
+    } 
+    catch (const CarAccident& accident) 
+    {
+        std::cerr << "An accident happened: " << accident.what() << std::endl;
```

Comment:
accident.what(). Here what() function is not clear to me by it's name. If you could provide other suitable name then it will be good.